### PR TITLE
Gui Live interface

### PR DESCRIPTION
**Pull request by tkolenbrander**
This pull request contains the basic UI for the live view. The interface should still be extended to the needs of the cameramen.

### CODE REVIEW

**Comment by Ettatabe** [comment]
This looks good and I approve it.

File: .classpath [hunk 1]
```diff
@@ -6,7 +6,7 @@
 			<attribute name="maven.pomderived" value="true"/>
 		</attributes>
 	</classpathentry>
-	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-1.8">
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER">
```
Comment (stevenmeijer) [inline .classpath]: This change should not be pushed. This repository should be forced to use java 1.8, since otherwise it won't work

**Comment by stevenmeijer** [comment]
Apart from the included classpath file this looks good

File: .classpath (hunk 1 above)
Comment (tkolenbrander) [inline .classpath]: You're right, I fixed it!

**Comment by TimYuenior** [comment]
Looks fine for merge now. Merging.

**Pull request merged**